CHANGELOG.md

```diff
@@ -2,6 +2,85 @@
 
 This document outlines major changes between releases.
 
+## 0.99.5 "Underestimation" (11 Nov 2022)
+
+It wasn't long since the previous release before a juicy set of features and
+optimisations came on board, so we've decided to change our release plan and
+supply one more 3.4.0 compatible version. It comes up with a massive suite
+of network P2P and broadcast improvements. We've carefully investigated message
+sending timeouts on benchmarks, reworked the broadcast logic and significantly
+reduced delays in message sending and amount of useless traffic between our
+nodes. For smart-contract developers and RPC client users several new features
+were added. These are automatic RPC bindings generator, poll-based and
+websocket-based transactions awaiting and extended web-socket client
+subscriptions interface. For better debugging experience we've improved state
+related functionality of VM CLI and added an ability to reset chain state to a
+particular height.
+
+An important dBFT initialization bug that caused timestamp-related errors on
+block addition was fixed. Also, those who use NeoGo node with
+EnableCORSWorkaround setting may want to update to be able to handle web-socket
+requests with Origin set in header. Finally, we're kindly asking our users to
+make acquaintance with our roadmap file where deprecated code removal schedule
+is placed.
+
+This version is fully compatible with C# node 3.4.0 and does not require
+resynchronization on upgrade. This time for sure, the next release is planned
+to be compatible with 3.5.0.
+
+New features:
+ * logarithmic gossip fan-out, new `BroadcastFactor` application setting and
+   adaptive peer requests are added to optimise broadcasting process (#608,
+   #2678, #2743)
+ * Prometheus histograms with handling time are added for P2P commands (#2750)
+   and RPC calls (#2784)
+ * transaction awaiting is added to RPC clients (#2740, #2749)
+ * RPC bindings generator (#2705) is added for both safe (#2766) and
+   state-changing (#2769, #2778) methods with initial iterator support (#2783)
+ * the ability to reset chain state to a particular height is added (#2576,
+   #2773)
+
+Behaviour changes:
+ * network packets sending is now limited in time (#2738)
+ * broadcast messages will be sent concurrently to each peer (#2741)
+ * ping messages are broadcast now (#2742)
+ * transactions requested by dBFT will be accepted on demand only (#2746, #2748,
+   nspcc-dev/dbft#63)
```

Comment:
As well as everything above, it's not something end user can immediately feel or need to take care about.